src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -54,6 +40,17 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.function.Function;
+import javax.annotation.Nullable;
+import lombok.SneakyThrows;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.apache.commons.validator.routines.InetAddressValidator;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.os.ExecutableFinder;
+import org.openqa.selenium.remote.Browser;
+import org.openqa.selenium.remote.service.DriverService;
```

Comment:
a lot of unnecessary changes in the import order

Reply:
Import order got changed as per alphanumeric order after I formatted the code as per the check-style recommended, should I change it back?

Reply:
Updated in the new commit